themes/jb/layouts/episode/single.html

```diff
@@ -1,7 +1,3 @@
-{{ define "headcontent" }}
-  <link href="https://vjs.zencdn.net/7.18.1/video-js.css" rel="stylesheet" />
-{{end}}
-
 {{ define "main" }}
```

Comment:
@devotoare, is there a reason why you moved the JS out of the single template?

I believe it was put there so it'd only load on that JS on the episode pages instead of every page.

You could still do the same thing you're doing with the `GetRemote` function w/Hugo, but just inside that template.

Based on my later feedback ([here](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/pull/445/files#r990674593) and [here](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/pull/445/files#r990675234)), this'll integrate everything into this one file. (if we end up putting it back in this file, if not this can be rejected and the other ones applied)

```suggestion
{{ define "headcontent" }}
  {{ $video := resources.GetRemote "https://vjs.zencdn.net/7.18.1/video-js.min.css" | fingerprint }}
  <link rel="stylesheet" href="{{ $video.RelPermalink }}" integrity="{{ $video.Data.Integrity }}">
{{end}}

{{ define "main" }}
```

I'd typically also like to do a [check for production](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/blob/a7585a36094491065cf88599b39c536eb68aad4e/themes/jb/layouts/partials/head.html#L12-L19), but since this is 3rd party code I don't think it's important to do that.